### DISCUSSION TITLE
Problem of rowhead and the number of rows in longtblr.

### DISCUSSION THREAD

**Discussion by note286** [lvjr/tabularray#264] [Q&A]
iIf `rowhead` is less than the number of rows in `longtblr`, an error will be appear. Is this a design or a bug? It is reasonable for me to design the format first and fill in the contents later. The current version cannot meet this requirement.

```latex
\documentclass{article}
\usepackage{tabularray}
\begin{document}
\begin{longtblr}{colspec={ll},rowhead=1}
AAA & BBB \\
% AAA & BBB \\
\end{longtblr}
\end{document}
```

In the above example, only one line will go wrong, but two lines won't.

**Comment by lvjr** [comment]
The `rowcount` counter comes in handy when `tabularray` is parsing the inner specifications:

```latex
\documentclass{article}
\usepackage{tabularray}
\begin{document}
\ExplSyntaxOn
\begin{longtblr}{colspec={ll},rowhead=\int_compare:nNnTF{\value{rowcount}}>{1}{1}{0}}
AAA & BBB \\
% AAA & BBB \\
\end{longtblr}
\ExplSyntaxOff
\end{document}
```

**Reply by note286** [reply]
It solved my question perfectly. Thank you very much.
